cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/permission/IPermissionManagement.java

```diff
@@ -271,4 +271,29 @@ default boolean tryExtendedGroups(IPermissionGroup permissionGroup, String group
 
         return false;
     }
+
+    default Collection<Permission> getAllPermissions(IPermissible permissible) {
+        return this.getAllPermissions(permissible, null);
+    }
+
+    default Collection<Permission> getAllPermissions(IPermissible permissible, String group) {
+        if (permissible == null)
+            return null;
```

Comment:
You can get an empty collection back.

Reply:
Yeah, returning an empty collection is the only correct answer here.

Reply:
fixed